File: basic-billing-api/BasicBilling.API/Entities/Billing.cs
using System;
using System.ComponentModel.DataAnnotations;

namespace BasicBilling.API
{
    public class Billing
    {
        [Key]
        public int Id { get; set; }

        public string Service { get; set; }        

        public string Period { get; set; }

        public string Status { get; set; }

        public string Client { get; set; }

    }
}

File: basic-service-payment/basic-service-payment/src/app/billing/shared/model/billing.ts
import { Client } from "src/app/client/shared/model/client";
import { Service } from "src/app/service/shared/model/service";
import { Currency } from "./currency";
import { Status } from "./status";

export interface Billing {
    id: Number;
    service: string;
    client: string;
    period: string;
    status: Status;
    currency: Currency;
}



